<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnUserData">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbConnectionStatus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>9</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connection Status : Disconnected</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDateTime">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>5</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time:_______            Date:________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../picture/image_tools/image.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnection">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbBorder">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>821</width>
     <height>9</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbConnectImage">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../picture/image_tools/Disconnect.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxConnection">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>821</width>
     <height>801</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="cbxBaudRateUart">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxScanPortUart">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lbBaudRate_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Baud Rate:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnectUart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScanUart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scan Port</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxUserData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>801</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBoxImg_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>401</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Image and ID</string>
     </property>
     <widget class="QGraphicsView" name="graphicsViewImageData_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>341</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="lbIDUser">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID :_______________________</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxDetailData_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>741</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Detailed Data</string>
     </property>
     <widget class="QLabel" name="lbName_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name            :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDispName">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDispAddress">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbAddress_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Address        :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDispCity">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCity_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>City               :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDispCountry">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Waiting...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCountry_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Country        :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearUserData">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>310</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbxImageID">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>801</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="lb_name_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtAddress">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_address_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtCity">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_city_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtCountry">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>651</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_country_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxRecordViewRecord_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>781</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Record View</string>
      </property>
      <widget class="QLabel" name="lb_search_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Search Here:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textSearch_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="rioSearchName">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search by name</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rioSearchID">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search by ID</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsViewImage_10">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>70</y>
         <width>191</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidgetRecord">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>531</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnSaveUser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClrDataInput">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxGetID_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Get ID</string>
      </property>
      <widget class="QPushButton" name="btnScanUser">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lbIDRegister">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID    _________</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxImage_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Image</string>
      </property>
      <widget class="QPushButton" name="btnImagePath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../picture/image_tools/Click_to_browse.png</normaloff>../picture/image_tools/Click_to_browse.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../picture/iconfinder_Computer__Computer_Hardware_USB_Port_Connection_4064143.png</pixmap>
      </property>
     </widget>
     <widget class="QGraphicsView" name="grapViewImgLoading">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCloseLoading">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../picture/image_tools/iconfinder_Button_Close-01_1221894.png</normaloff>../picture/image_tools/iconfinder_Button_Close-01_1221894.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lb_country_11">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>770</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbReadLoading">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reading Tag...</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbWaitLoading">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../picture/image_tools/805.gif</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_select">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../picture/image_tools/icon_select.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditData">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registration / Edit User Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
